docs/Rules.md

```diff
@@ -1353,6 +1353,7 @@ Rule2
 ------------------------------------------------------------------------------
 
 ### Arithmetic commands used with VAR
+Note: arithmetic is done using single point precision floating point.  This means calculations involving values larger than approx. 16 million (ex: `%utctime%`) will not be precise.
```

Comment:
Correct english would be "will not be accurate"
Thanks

Reply:
In general English, yes, but in the context of math precise and accurate are different things.  In this case the result _will_ be accurate, just not precise.